ReleaseTooling/Sources/FirebaseManifest/FirebaseManifest.swift

```diff
@@ -53,7 +53,6 @@ public let shared = Manifest(
     Pod("FirebasePerformance", platforms: ["ios", "tvos"], zip: true),
     Pod("FirebaseStorage", zip: true),
     Pod("FirebaseMLModelDownloader", isBeta: true, zip: true),
-    Pod("FirebaseVertexAI", isBeta: true, allowWarnings: true, zip: true),
```

Comment:
My preference would be commented out with a TODO - since we've tested this version.

Reply:
Good call. Done.